New Spanish language added via upload

antoromani opened the pull request:
New Spanish language added, please review

antoromani committed:
Add files via upload

New Spanish language added

Vovan-VE commented on locales/es/front/credits.json:
```diff
@@ -0,0 +1,95 @@
+      "LinkToPlanItem": "{date}: {paid} {ofTotal} ({percent}%)",
+      "NoLinkToPlan": "fuera del plan",
+      "SelectTransaction": "Seleccione transacción...",
+      "TransactionLinked": "Transacción vinculada por {amount} en {date}{ifAccountValid, seleccione, 0{ with another account} otra{}}",
```
Please, check ICU `{select}` syntax: https://formatjs.io/docs/core-concepts/icu-syntax/#select-format
There are several `{select}`s is all the files.

antoromani replied:
Hello, I made the corrections.

Vovan-VE commented on locales/es/front/receipts.json:
```diff
@@ -0,0 +1,60 @@
+      "ConfirmDeleteReceipt": "¿Está seguro de eliminar este recibo?",
+      "LinkedTransactionsWillLeft": "Las transacciones vinculadas ({count}) no se eliminarán.",
+      "NoPositions": "Sin posiciones",
+      "SelectPositionsToTrack": "Seleccione {maxCount, plural, =1{a position} uno{up to # position} otro{up to # positions}} para ver precios."
```
Check also `{plural}` syntax: https://formatjs.io/docs/core-concepts/icu-syntax/#plural-format
There are several `{plural}`s is all the files.

antoromani replied:
I also reviewed 4 more files. Thank you for the indications

Vovan-VE reviewed (changes requested):
It's my fail to not mention ICU syntax in README. Sorry for that.

In general, all the "keys" in `select` and `plural` must be untouched, and all of then must include `other` key, even if empty `other{}`, or formatting will raise an error. Check any existing translations for reference (same filename same line number), better non-familar language.

antoromani committed:
Reviewed Files

Hello, I made the corrections. I also reviewed 4 more files. Thank you for the indications

Vovan-VE commented:
@antoromani , Thanks for contribution! Sorry for delay, I will check it as soon as possible.

Vovan-VE commented:
@antoromani , language published. Thanks for contribution! Feel free to collect remarks, if you will find some, to open new PR later.
